ts/Series/Treegraph/TreegraphSeries.ts

```diff
@@ -430,84 +500,106 @@ class TreegraphSeries extends OrganizationSeries {
      * @private
      */
     public pointAttribs(
-        point: TreegraphPoint | TreegraphNode,
-        state: StatesOptionsKey
+        point: TreegraphPoint,
+        state?: StatesOptionsKey
     ): SVGAttributes {
-        if (point && point.isNode) {
-            const { opacity, ...attrs } = Series.prototype.pointAttribs.apply(
-                this,
-                arguments
+        const series = this,
+            attribs = Series.prototype.pointAttribs.call(series, point, state),
+            levelOptions =
+                (series.mapOptionsToLevel as any)[point.node.level || 0] || {},
+            options = point.options,
+            stateOptions =
+                (levelOptions.states &&
+                    (levelOptions.states as any)[state as any]) ||
+                {},
+            borderRadius = pick(
+                stateOptions.borderRadius,
+                options.borderRadius,
+                levelOptions.borderRadius,
+                series.options.borderRadius
+            ),
+            linkColor = pick(
+                stateOptions.link && stateOptions.link.color,
+                options.link && options.link.color,
+                levelOptions.link && levelOptions.link.color,
+                series.options.link && series.options.link.color
+            ),
+            linkLineWidth = pick(
```

Comment:
I am sure we may make this code smaller, e.g. using updated strings array and forEach.